src/pages/Systems/MachineInfo.tsx

```diff
@@ -83,31 +84,50 @@ const InfoItem = (props: { title: string; value: string; width?: string; loading
 };
 
 const IngestorInfo = () => {
+	const [openDeleteModal, setOpenDeleteModal] = useState(false);
 	const [recentRecord] = useClusterStore((store) => store.currentMachineRecentRecord);
 	const [ingestorMachines] = useClusterStore((store) => store.ingestorMachines);
 	const ingestor = _.find(ingestorMachines, (ingestor) => ingestor.domain_name === recentRecord?.address);
 	const error = ingestor ? ingestor.error : null || null;
+	const deleteUrl = recentRecord?.address ?? '';
```

Comment:
use more appropriate variable name.

Reply:
recentRecord is already returning the currentMachineRecentRecord.